log/handler_glog.go

```diff
@@ -201,9 +197,10 @@ func (h *GlogHandler) Handle(_ context.Context, r slog.Record) error {
 				h.siteCache[r.PC], lvl, ok = rule.level, rule.level, true
 			}
 		}
-		// If no rule matched, remember to drop log the next time
+		// If no rule matched, use the default log lvl
 		if !ok {
-			h.siteCache[r.PC] = 0
+			lvl = slog.Level(h.level.Load())
+			h.siteCache[r.PC] = lvl
```

Comment:
If we set vmodule `GlogHandler.override` will be set, so `Handle` will always be called.  If the global verbosity is changed in the console after a log is emitted for the first time, when we go to evaluate whether to call the underlying Handler, we will use the cached log level.

To see how this is problematic, start geth in dev mode with arbitrary vmodule and high verbosity: `geth --dev --vmodule "accounts=12" --verbosity 5 console`.  In the console, set the global verbosity low.  Log statements from the higher verbosities will still be emitted.

Reply:
Added a commit to address that. It just clears the cache on setting the verbosity which will force a recalculation